packages/main/src/resolvers/conversation.ts

```diff
@@ -133,7 +134,7 @@ export const ConversationMutations: ConversationMutationsResolvers = {
       }
     }
 
-    return C.mustGetConversation(conversationId)
+    return thread
```

Comment:
Why did you change this? If you don't re-fetch the conversation from the cache then the client will not get the updated flag state.